ui/page/root/wallet_setup_page.go

```diff
@@ -627,5 +630,5 @@ func (pg *CreateWallet) validRestoreWalletInputs() bool {
 }
 
 func (pg *CreateWallet) walletCreationSuccessCallback() {
-	pg.ParentNavigator().Display(NewHomePage(pg.Load))
+	pg.ParentNavigator().Display(NewWalletSelectorPage(pg.Load))
 }
```

Comment:
Creating the first wallet on a new setup no longer shows the home page as expected.

Reply:
resolved